app/CustomFields/HeaderDefinition.php

```diff
@@ -83,15 +122,21 @@ public static function get($post_type) {
             'parallax' => 'Enable parallax effect',
             'carousel-buttons' => 'Enable previous/next carousel buttons (if used)',
             'scroll-down-button' => 'Enable scroll down button (if full-height is used)',
-            'display-post-category' => 'Display post category'
+            'display-post-category' => 'Display post category',
+            'autoplay' => 'Enable autoplay when the video is in view',
```

Comment:
Autoplay, loop, mute etc are all video hero specific, we should split these options up in to a separate ACF field which conditionally shows when video hero is selected